packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.ts

```diff
@@ -31,37 +33,37 @@ export const defaultState: FieldPluginData = {
   token: undefined,
   uid: '-preview',
   spaceId: undefined,
+})
+
+export type CreatePluginActionsOptions<Content> = {
+  uid: string
+  postToContainer: (message: unknown) => void
+  onUpdateState: (state: FieldPluginData<Content>) => void
+  parseContent: (content: unknown) => Content
 }
 
-export type CreatePluginActions = (
-  uid: string,
-  postToContainer: (message: unknown) => void,
-  onUpdateState: (state: FieldPluginData) => void,
-) => {
+export const createPluginActions = <Content>(
+  options: CreatePluginActionsOptions<Content>,
+): {
   // These functions are to be called by the field plugin when the user performs actions in the UI
-  actions: FieldPluginActions
+  actions: FieldPluginActions<Content>
   // These functions are called when the plugin receives messages from the container
   messageCallbacks: PluginMessageCallbacks
-}
-
-export const createPluginActions: CreatePluginActions = (
-  uid,
-  postToContainer,
-  onUpdateState,
-) => {
+} => {
+  const { uid, postToContainer, onUpdateState, parseContent } = options
   // Tracks the full state of the plugin.
   //  Because the container doesn't send the full state in its messages, we need to track it ourselves.
   //  isModal and height is not included in the messages to the children and must thus be tracked here.
   //  In future improved versions of the plugin API, this should not be needed.
-  let state: FieldPluginData = defaultState
+  let state: FieldPluginData<Content> = defaultState(parseContent(undefined))
```

Comment:
One could question whether it is legitimate to use `undefined` as a default value, but this version is more explicit than the old version. (It will get replaced as soon as a `loaded` message is received`, so this value will never be rendered)